paasta_tools/chronos_serviceinit.py

```diff
@@ -170,8 +170,11 @@ def _format_parents_verbose(job):
     parent_service_instances = [tuple(chronos_tools.decompose_job_id(parent)) for parent in parents]
 
     # find matching parent jobs
-    parent_jobs = [chronos_tools.get_job_for_service_instance(*service_instance)
-                   for service_instance in parent_service_instances]
+    parent_jobs = [
+        chronos_tools.get_jobs_for_service_instance(
+            *service_instance, include_disabled=False, include_temporary=False)[0]
```

Comment:
``include_disabled`` needs to be True here - It's fine for a parent job to be disabled, so we don't want to hide them in the output of status. It's just the temporary jobs we want to skip.